examples/gno.land/r/demo/markdown_test/markdown.gno

```diff
@@ -175,6 +173,403 @@ If markdown is too limiting, you can just insert your own <strike>crazy</strike>
 <div style='font-family: "Comic Sans MS", "Comic Sans", cursive;'>
 It is a pity, but markdown does **not** work in here for most markdown parsers.
 [Marked] handles it pretty well.
-</div>`
+</div>
+
+Components
+==========
+
+Jumbotron component
+-------------------
+
+A Jumbotron is a layout component used to showcase key messages.
+
+
+:::jumbotron
+
+## Jumbotron title
+
+This is a Jumbotron component you can fill with regular markdown.
+
+:::jumbotron/
+
+
+Stack component
+---------------
+
+Stack is a layout component used to group elements together and apply a space between them. For example, it is usefull with grouped links.
+
+:::stack
+[Stack link 1](/)
+[Stack link 2](/)
+:::stack/
+
+Box component
+-------------
+
+Box is the most abstract component that renders a div element. It is used as a group component.
+
+:::box
+**This content is in a box**
+:::box/
+
+
+Columns component
+-----------------
+
+Columns is a layout for managing columns layouts. It takes the numb of cols you want (*from 1 to 6 columns*) to create as argument and need to be filled with as much content as you mentioned in this argument. You can manage multiple element by grouping them in box components. 
+
+*Nested columns are not currently allowed*
+
+:::columns (2)
+:::box
+
+### Col 1
+
+Content 1
+
+:::box/
+:::box
+
+### Col 2
+
+Content 2
+
+:::box/
+:::columns/
+
+:::columns (3)
+:::box
+
+### Col 1
+
+Content 1
+
+:::box/
+:::box
+
+### Col 2
+
+Content 2
+
+:::box/
+:::box
+
+### Col 3
+
+Content 3
+
+:::box/
+:::columns/
+
+:::columns (4)
+:::box
+
+### Col 1
+
+Content 1
+
+:::box/
+:::box
+
+### Col 2
+
+Content 2
+
+:::box/
+:::box
+
+### Col 3
+
+Content 3
+
+:::box/
+:::box
+
+### Col 4
+
+Content 4
+
+:::box/
+:::columns/
+
+:::columns (5)
+:::box
+
+### Col 1
+
+Content 1
+
+:::box/
+:::box
+
+### Col 2
+
+Content 2
+
+:::box/
+:::box
+
+### Col 3
+
+Content 3
+
+:::box/
+:::box
+
+### Col 4
+
+Content 4
+
+:::box/
+:::box
+
+### Col 5
+
+Content 5
+
+:::box/
+:::columns/
+
+:::columns (6)
+:::box
+
+### Col 1
+
+Content 1
+
+:::box/
+:::box
+
+### Col 2
+
+Content 2
+
+:::box/
+:::box
+
+### Col 3
+
+Content 3
+
+:::box/
+:::box
+
+### Col 4
+
+Content 4
+
+:::box/
+:::box
+
+### Col 5
+
+Content 5
+
+:::box/
+:::box
+
+### Col 6
+
+Content 6
+
+:::box/
+:::columns/
+
+Button component
+----------------
+
+Button component is used as CTA (for a link) or to display a button element that you will be able to use to trigger an action or event through Javascript.
+
+:::button (https://gno.land)
+Link button
+:::button/
```

Comment:
```suggestion
:::button(https://gno.land,Link button)
```

```suggestion
[Link button](https://gno.land) :::button
```

What do you think about supporting inline sections for some simple components while maybe keeping the multiline an option for advanced cases?

Reply:
Inline components where already available with another style which is:

```
:::button(Plain button)/
```

In this example, the only difference with your proposition is that we need a trailing slash that is needed for few reason:
- Easier to avoid make a mess with marked-js plugin system (we use to extend markdown) with scoped components.
- Easier for human to understand that is a single line component and not the start of block component that will be close later in the code.

_I've updated buttons to be inline component by the way._